src/EnergyPlus/MixedAir.hh

```diff
@@ -154,8 +159,7 @@ namespace MixedAir {
         std::string Name;
         std::string ControllerType;
```

Comment:
Probably don't need `ControllerType` (and can rename `ControllerType_Num` to `controllerType`).

Reply:
I should know this, but I'll ask anyway. 
`std::string ControllerType` is used frequently in error messages (probably the only way it's used). 
So we'd rather write the error message with `MixedAirControllerTypeNames[static_cast<int>(ControllerType)]` than storing the string on the controller struct?

Reply:
Yes and yes.